perun-base/src/main/java/cz/metacentrum/perun/audit/events/MailManagerEvents/InvitationSentEvent.java

```diff
@@ -0,0 +1,62 @@
+package cz.metacentrum.perun.audit.events.MailManagerEvents;
+
+import cz.metacentrum.perun.audit.events.AuditEvent;
+import cz.metacentrum.perun.core.api.Group;
+import cz.metacentrum.perun.core.api.User;
+import cz.metacentrum.perun.core.api.Vo;
+
+public class InvitationSentEvent extends AuditEvent {
+
+	private User sender;
+	private String email;
+	private String language;
+	private String message;
+	private Vo vo;
+	private Group group;
+
+	@SuppressWarnings("unused") // used by jackson mapper
+	public InvitationSentEvent() {
+	}
+
+	public InvitationSentEvent(User user, String mail, String language, Group group, Vo vo) {
+		this.sender = user;
+		this.email = mail;
+		this.language = language;
+		this.group = group;
+		this.vo = vo;
+
+		this.message = formatMessage("Invitation to VO: %s, group: %s, sent by user: %s, to email: %s, in language: %s",
+			vo, group == null ? "None" : group, sender, email, language);
```

Comment:
You don't have to pass string "None", group will resolve to null in resulting message. Since we still sometimes parse/check messages, you should probably omit ``VO:`` and ``group:`` before objects, since they will be resolved to something like ``VO: Vo:[....]``.
I suggest to modify message based on passed VO/group: 
```java
formatMessage("Invitation to %s, sent by %s, to email: %s, in language: %s.", (group != null) ? group : vo, sender, email, language);
```